### PR TITLE
Adding a simple send `hello world` message to a channel

### PR DESCRIPTION
**Pull request by awls99**
There's no simple hello world example for simple sending of messages, thought it might help other people.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,16 @@ bot.run
 
 This bot responds to every "Ping!" with a "Pong!".
 
+Or you can simply send a message to your chanel like this:
+
+```ruby
+require 'discordrb'
+
+bot = Discordrb::Bot.new token: '<token>', client_id: 168123456789123456
+bot.run true
```
Comment (z64) [inline README.md]: Running `async` here is pointless and doesn't really have much to do with this example. I would rather not encourage people to use this if they don't understand what they're doing. Please remove `true` and move it to the end of the file.

**Comment by VxJasonxV** [comment]
Is this better suited as an [example file](https://github.com/meew0/discordrb/tree/master/examples)? The README is illustrating a series of common features (bot initialization, eventing, responding), but isn't a way to teach you how to use the library.

File: README.md (hunk 1 above)
Comment (awls99) [inline README.md]: Fair enough, I was using it because I wanted to post a message and stop.

Comment (z64) [inline README.md]: Right - in which case, you can just completely omit `bot.run`. `send_message` just makes a REST call and does not need to initiate a gateway connection.

**Comment by awls99** [comment]
>If you would like to add a send_message.rb there, and include other examples of sending messages (i.e. Channel#send_message, etc.) then this might be worth having.

Yes this, would be better in the examples, but  it would imply me knowing how, if I ever get around that I will add examples then

**Pull request closed without merging**